app/assets/javascripts/repositories.js

```diff
@@ -0,0 +1,16 @@
+function showField(){
+	document.getElementById("password").style.display = 'block'
+}
+
+function hideField(){
+	$("#change_pass").hide();
+	document.getElementById("password").style.display = 'none'
+}
+function toggleField() {
+		var x = document.getElementById("password")
+		if (x.style.display === 'none') {
+			document.getElementById("password").style.display = 'block'

```

Comment:
Can you do `x.style.display = 'block'` instead of getting the element again?

Reply:
done